src/integration-test/java/org/ow2/proactive/catalog/service/CatalogObjectServiceIntegrationTest.java

```diff
@@ -417,6 +417,45 @@ public void testCatalogObjectWithProjectNameAsQueryParamAndInMetadataListAndInWo
 
     }
 
+    @Test
+    public void testRestoreCatalogObject() throws IOException, InterruptedException {
+        String workflowName = "workflow";
```

Comment:
is it workflowName or object kind ?

Reply:
It is a kind

Reply:
it'll be more clear to rename it "objectKind"